lib/resources/r4/procedure.yaml

```diff
@@ -33,19 +47,217 @@ fields:
       - display: ICD-10-PCS
         system: https://www.icd10data.com/icd10pcs
         info_link: https://www.icd10data.com/icd10pcs
+  example: |
+    {
+      "coding": [
+        {
+          "system": "http://snomed.info/sct",
+          "code": "119954001",
+          "display": "Adenoid excision (procedure)",
+          "userSelected": true
+        }
+      ],
+      "text": "Adenoid excision (text)"
+    }
+  note: |
+    <ul>
+      <li> Must be provided. </li>
+    </ul>
+
+- name: resourceType
```

Comment:
Can you order these better? I would prefer the way they are listed on hl7. AKA resourceType should be first.

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/372/commits/a9443bbbf959cd765f514708b4c1c11aeec9b263).